config/service_providers.yml

```diff
@@ -161,6 +161,38 @@ production:
     attribute_bundle:
       - email
 
+# Micro-purchase
+  'urn:gov:gsa:saml:2.0.profiles:sp:sso:micropurchase-staging':
+    acs_url: 'https://micropurchase-staging.18f.gov/auth/saml/callback'
+    assertion_consumer_logout_service_url: 'https://micropurchase-staging.gov/auth/saml/logout'
+    sp_initiated_login_url: 'https://micropurchase-staging.18f.gov/login'
+    block_encryption: 'aes256-cbc'
+    cert: 'sp_micropurchase'
+    agency: 'TTS Acquisition'
+    friendly_name: 'Micro-purchase Staging'
+    logo: 'generic.svg'
```

Comment:
if you want to include an image, say something like:
<img width="282" alt="screen shot 2017-01-10 at 10 53 00 am" src="https://cloud.githubusercontent.com/assets/1328699/21820045/045758d2-d723-11e6-9f66-28d3aa6b83c4.png">

just include the file [here](https://github.com/18F/identity-idp/tree/master/app/assets/images/sp-logos) and reference the filename